UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -98,6 +104,16 @@ private void Update()
                 ref turnSmoothSpeed,
                 turnSmoothTime);
         }
+
+        if (!(Vector3.Angle(Vector3.up, hitNormal) <= slopeLimit))
```

Comment:
A few comments here:

1. If you're doing `if(a) b = false; else b = true;` you might as well write `b=a`. 
2. The negating is quite confusing, why say `not smaller than` instead of `bigger than`?

Instead of these 10 lines i'd write:

`canJump = Vector3.Angle(Vector3.up, hitNormal) <= slopeLimit;`

Functionally equivalent, concise and (in my opinion) more readable.